Sources/Basic/TerminalController.swift

```diff
@@ -132,30 +132,28 @@ public protocol ProgressBarProtocol {
 }
 
 // Simple ProgressBar which shows the update text in new lines.
-public final class SimpleProgressBar<Target: OutputStream>: ProgressBarProtocol {
+public final class SimpleProgressBar: ProgressBarProtocol {
     private let header: String
     private var isClear: Bool
-    private var stream: UnsafeMutablePointer<Target>
+    private var stream: OutputByteStream
```

Comment:
@ddunbar I am not sure which is better here, I think this operating on `OutputByteStream` makes more sense but a generic `OutputStream` will allow us to stream to things like strings too. I'll fixup or remove this commit once you review.